fitlins/workflows/base.py

```diff
@@ -57,17 +57,16 @@ def init_fitlins_wf(bids_dir, derivatives, out_dir, analysis_level, space,
         name='getter')
 
     if smoothing:
-        smoothing_params = smoothing.split(':', 1)
-        if smoothing_params[0] != 'iso':
-            raise ValueError(f"Unknown smoothing type {smoothing_params[0]}")
-        smoothing_fwhm = float(smoothing_params[1])
+        smoothing_params = smoothing.split(':', 2)
```

Comment:
I would suggest that we make `run` the default, so we interpret `iso:5` as `run:iso:5`. This makes it backwards compatible.

```suggestion
        smoothing_params = smoothing.split(':', 2)
        if len(smoothing_params) < 3:
            smoothing_params.insert(0, 'run')
```

Reply:
If so, wouldn't it make more sense to make it `TYPE:FWHM:LEVEL` ?

Reply:
I see what you're saying, but `LEVEL:TYPE:FWHM` feels like the order of significance to me. Do you disagree?